nexus/src/app/sled.rs

```diff
@@ -115,7 +116,7 @@ impl super::Nexus {
         opctx: &OpContext,
         pagparams: &DataPageParams<'_, Uuid>,
     ) -> ListResultVec<db::model::Sled> {
-        self.db_datastore.sled_list(&opctx, pagparams).await
+        self.db_datastore.sled_list(&opctx, pagparams, SledFilter::All).await
```

Comment:
I'm not sure where this is called, but from context I would have guessed this should be `::InService` (e.g., why would Nexus list expunged sleds?). Or maybe this method could take a filter as an argument?

Reply:
Hmm. I mainly was sticking with the existing functionality here, but I think you are correct. I'll change this.

Reply:
Fixed in 8b07fa885bdf94ed2009e1bda4da5d34dd6b4c72